SU-HSDO/suhumsci
# SHS-5630: Editors can more clearly see what a change to media will affect

# READY FOR REVIEW

## Summary
When editing a media entity that is being used someplace, it now shows a link (which opens in a new tab) in the warning message to better allow an editor to see usage.

![Screenshot 2024-07-23 at 6 29 04 PM](https://github.com/user-attachments/assets/2359e525-3cbf-41f4-912a-3d786647cd8d)

## Need Review By (Date)
2024-07-29

## Urgency
low

## Steps to Test
1. Login to the site and visit the media overview page at `/admin/content/media`
2. Click "Edit" next to one of the pieces of media
3. If that media is being used, an orange warning box will show up on top of the form (see screenshot above)
4. If there is no warning box, try another piece of media
5. Verify that the link that says "xxx pieces of content" takes you to the media usage page in a new tab.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207890476312472
